iModelCore/ECPresentation/Source/Content/ContentItemBuilder.h

```diff
@@ -153,7 +153,7 @@ struct ContentItemBuilder
     ContentSetItemExtendedData& GetExtendedData() { return m_extendedData; }
     ContentSetItemExtendedData const& GetExtendedData() const { return m_extendedData; }
 
-    void AddValue(Utf8CP name, Utf8CP rawAndDisplayValue, ECPropertyCP);
+    void AddValue(Utf8CP name, IECSqlValue const& value, PrimitiveType type);
     void AddValue(Utf8CP name, ECPropertyCR ecProperty, IECSqlValue const& value);

```

Comment:
for consistency between the two `AddValue` functions, I think `PrimitiveType` should come first and `IECSqlValue` should come second

Reply:
Changed